ci_framework/playbooks/ceph.yml

```diff
@@ -159,6 +161,22 @@
       loop: "{{ lookup('file', cifmw_cephadm_spec_ansible_host) | from_yaml_all | list }}"
       delegate_to: localhost
 
+    - name: Get already assigned IP addresses
+      ansible.builtin.set_fact:
+        ips: "{{ ips | default([]) + [ hostvars[item][all_addresses] | ipaddr(storage_network_range) | first ] }}"
```

Comment:
I thought this was already defined on line 95. Is that fact really unavailable here?

Reply:
It's unavailable because they are different playbook, and the scope of the facts are at playbook level:

```
TASK [DEBUG msg={{ host_to_ip }}] *******************************************************************************************************************************************************************************************************************************************************
Wednesday 06 September 2023  02:11:55 -0400 (0:00:00.080)       0:00:10.754 ***
fatal: [192.168.130.20]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'host_to_ip' is undefined. 'host_to_ip' is undefined\n\nThe error appears to be in '/root/ci-framework/ci_framework/playbooks/ceph.yml': line 169, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n      loop: \"{{ groups['edpm'] }}\"\n    - name: DEBUG\n      ^ here\n"}
```

This is the reason I rebuilt that IPs list on L166. Do you know if there's a way to let the fact "survive" on different playbook executions? Otherwise I guess it's fair enough getting it here.